src/main/resources/application.properties

```diff
@@ -5,15 +5,15 @@ spring.jackson.default-property-inclusion=non_null
 # Example if build.version=1.2.3-internal becomes Version 0.0.18 (1.2.3-internal)
 build.version=
 
-server.port=8080
+server.port=8089
 #server.ssl.key-store: <keystore.p12>
 #server.ssl.key-store-password: <mypassword>
 #server.ssl.keyStoreType: <PKCS12>
 #server.ssl.keyAlias: <tomcat>
 
 ######## EI Front-End
 ei.frontendServiceHost=localhost
-ei.frontendServicePort=8080
+ei.frontendServicePort=8090
```

Comment:
These two need to be the same port. One is to specify which port the front end will start on and the other is used by the frontend to know where the bridge is.

Reply:
I think we should use default 8080 instead of 8090,,, since we have port 8080 default in other Eiffel. components.

Reply:
Done